modules/visual_script/visual_script.cpp

```diff
@@ -808,17 +711,16 @@ void VisualScript::get_custom_signal_list(List<StringName> *r_custom_signals) co
 }
 
 int VisualScript::get_available_id() const {
-	int max_id = 0;
-	for (Map<StringName, Function>::Element *E = functions.front(); E; E = E->next()) {
-		if (E->get().nodes.empty()) {
-			continue;
+	// TODO: Maybe add a class instance counter here to get the available id??
```

Comment:
What's the status here?

Reply:
Oh I decided against that as that seemed wasteful. Will remove all the useless TODO comments. :)